composer/loggers/mlflow_logger.py

```diff
@@ -145,6 +149,26 @@ def log_hyperparameters(self, hyperparameters: Dict[str, Any]):
             )
             self._optimized_mlflow_client.flush(synchronous=False)
 
+    def log_images(
+        self,
+        images: Union[np.ndarray, torch.Tensor, Sequence[Union[np.ndarray, torch.Tensor]]],
+        name: str = 'image',
+        channels_last: bool = False,
+        step: Optional[int] = None,
+        masks: Optional[Dict[str, Union[np.ndarray, torch.Tensor, Sequence[Union[np.ndarray, torch.Tensor]]]]] = None,
+        mask_class_labels: Optional[Dict[int, str]] = None,
+        use_table: bool = True,
+    ):
+        del masks, mask_class_labels, use_table  # Unused (only for wandb)
```

Comment:
should we raise if these are set? silently not doing segmentation logging seems not great.

Reply:
added warning.